app/components/SCForm.js

```diff
@@ -96,9 +96,12 @@ class SCForm extends Component {
   onChange(value) {
     const formInfo = this.props.navigation.state.params.form;
     const length = formInfo.fields.length;
-    let newState = {};
     let fieldValue, fieldKey, max, min;
     let fieldValidations = [];
+    let newState = ({
+      ...this.state,
+      field_key: fieldValue
+    });
```

Comment:
I was planning to use newState to check the value of the old State verses the new. Don't think it's relevant any longer.